### DISCUSSION TITLE
Option in the scripts that just install the wanted application.

### DISCUSSION THREAD

**Discussion by Ansichtsritter** [tteck/Proxmox#3780] [General]
Hello,

i really like the scripts. In the meantime, however, I prefer to use VMs instead of LXC - I now prefer the complete isolation of the systems. Therefore, I would like to know if there are plans or possibilities to give the scripts an option to install only the desired program. So I don't run the script on the PVE and a LXC is pulled up in which the program is installed instead, but in a provided VM and only the program is installed accordingly.

Regards

Ansichtsritter

**Comment by MickLesk** [comment]
Hi,

you would be better served by looking at the scripts and copy-pasting the lines of code. Everything else is a massive overhead (scan which VM's are there, select VM etc.). 
I'm also not sure whether Proxmox supports this well enough.
